tests/pester-tests/features/CredentialProvider.Tests.ps1

```diff
@@ -0,0 +1,63 @@
+﻿Describe 'Ensuring credentials do not bleed from configured sources' -Tag CredentialProvider -ForEach @(

```

Comment:
I personally think that we need some additional words here as to exactly that this test is trying to assert.

Why are we expecting 0 ExitCodes for some commands, and not for others?

If we are expecting that Chocolatey CLI should prompt for credentials, since it is no longer able to be provided credentials from another defined source, why is `choco outdated` not returning an error exit code?  Same question goes for `choco info`.

I think having the extra description about what this test is trying to assert would help to understand this test, without having to read through the whole thing.

Reply:
I've updated this to have some comments about what it's testing and why the exit codes for info and outdated are 0. I also re-ordered them to put info and outdated beside each other to avoid duplicating the comment.